src/router.ts

```diff
@@ -1140,6 +1141,31 @@ export async function handleSlackVoterThreadMessage(
     slackRetryNum: retryCount,
     slackRetryReason: retryReason,
   });
+  outboundDbMessageEntry.slackFiles = reqBody.event.files;
+
+  // check attachments
+  if (outboundDbMessageEntry.slackFiles) {
+    const errors = MessageParser.validateSlackAttachments(
+      outboundDbMessageEntry.slackFiles
+    );
+    if (errors.length) {
+      await SlackApiUtil.sendMessage(
+        'Sorry, there was a problem with one or more of your attachments:\n' +
```

Comment:
Is there a way to test that this works correctly (and if so, can you add a screenshot to the pull request description)?

Reply:
![image](https://user-images.githubusercontent.com/433031/97010711-66e46600-150b-11eb-8509-14d941f97f1a.png)